pages/_slug/index.vue

```diff
@@ -11,10 +11,14 @@ export default {
 		PageArticle
 	},
 	data () {
-		const currentPage = this.$store.state.sitePages.filter(oneSitePage => oneSitePage.slug === this.$route.params.slug)[0];
-		return {
+		const currentPage = this.$store.state.sitePages.find(oneSitePage => oneSitePage.slug === this.$route.params.slug);
+
+		return currentPage ? {
 			title: currentPage.title,
 			content: currentPage.content
+		} : {
+			title: "Not Found",
```

Comment:
This stops the website from rendering the `layouts/error.vue` page

Reply:
Good catch, @TeddTech. This is fixed in the new commit. Thanks.